[devtools] Update network filtering reference (#9024)
google/WebFundamentals: src/content/en/tools/chrome-devtools/network/reference.md

Below is a complete list of supported properties.

> * `cookie-domain`. Show the resources that set a specific [cookie domain](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#Directives)..

Reviewer: ```suggestion
* `cookie-domain`. Show the resources that set a specific [cookie domain](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#Directives).
```

* `cookie-name`. Show the resources that set a specific [cookie name](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#Directives).
* `cookie-path`. Show the resources that set a specific [cookie path](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#Directives).
* `cookie-value`. Show the resources that set a specific [cookie value](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#Directives).
* `domain`. Only display resources from the specified domain. You can use
  a wildcard character (`*`) to include multiple domains. For example, `*.com`

>   displays resources from all domain names ending in `.com`. DevTools

Reviewer: words missing?

```suggestion
  displays resources from all domain names ending in `.com`. DevTools shows a
```

  dropdown with all MIME types it has encountered.
* `mixed-content`. Show all mixed content resources (`mixed-content:all`) or
  just the ones that are currently displayed (`mixed-content:displayed`).

> * `priority`. xxx.

Reviewer: TODO

* `status-code`. Only show resources whose HTTP status code match the

>   specified code. DevTools populates the autocomplete dropdown menu with all

Reviewer: Let's refer to either "dropdown" (line 281) or "autocomplete dropdown menu" consistently (pick one)

  of the status codes it has encountered.

> * `url`. Show the resources that have a `url` with a value that matches the specified value.

Reviewer: simpler:

```suggestion
* `url`. Show the resources that have a `url` matching the specified value.
```
